lib/rex/ui/text/dispatcher_shell.rb

```diff
@@ -417,73 +417,6 @@ def tab_complete_helper(dispatcher, str, words)
     return items
   end
 
-  # Processes a resource script file for the console.
```

Comment:
Don't forget to `include Resource` at the top here too, since you're removing the function.

Reply:
T_T, I am so careless... It's all my fault, I think the problem is solved~ I just tested the new changes, thank you @timwr really so much for your gentle help.